doc/swagger.json

```diff
@@ -97,11 +97,11 @@
             "description": "Password",
             "type": "string"
           },
-          "uiLg": {
+          "userLanguage": {
```

Comment:
It should be marked as required. What do you think?

Reply:
You're right, I'll update the documentation :+1: